### PR TITLE
This week in information security (week of June 22nd)

### PR DESCRIPTION
**Pull request by cneill**
__Bio:__
Charles Neill is a Security Engineer at Rackspace, who tests Rackspace products as well as OpenStack projects like Barbican. He is also a developer on [DefectDojo](https://github.com/rackerlabs/django-DefectDojo), an open source defect tracking system using the Django framework.

__Social Media:__
Twitter: [@ccneill](https://twitter.com/ccneill)
Github: [cneill](https://github.com/cneill)
Website: [inventropy.us/blog/](https://inventropy.us/blog/)

__Editing Preference:__
Feel free to edit as needed.

__Posting Preference:__
Desired date to publish: June 25th, 2015

Thank you!

### CODE REVIEW

**Comment by ktbartholomew** [comment]
@catlook do you have any edits for Charles?

**Comment by catlook** [comment]
checking now...thanks for the reminder!!

-- Cat Lookabaugh, Sr. Information Developer
Rackspace Hosting - the #1 Managed Cloud Company
DevDocs on Developer Experience Team
Office: (210) 312-8193   Cell: (210) 364-1060
cat.lookabaugh@rackspace.com

We should not write so that it is possible for the reader to understand us, but so that it is impossible for him to misunderstand us. - Quintilian, rhetorician (c. 35-100)

Harmony + Learner + Responsibility + Context + Intellection + Arranger + Consistency + Relator + Input + Analytical

File: _posts/2015-06-22-this-week-in-infosec-jun-22.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+As always, you can find me on Twitter [__@ccneill__][twitter] if you have any thoughts on this post.
+
+<!-- more -->
+
+# News / Opinions
+
+- [__LastPass Hacked__][lastpass] - In a post on their blog released last week, LastPass admitted that their servers had been compromised, exposing users' "email addresses, password reminders, server per user salts, and authentication hashes." Due to their usage of [PBKDF2-SHA256][pbkdf2] with 100k rounds, this compromise isn't as bad as it could be; cracking the obtained hashes will be harder than if something like vanilla MD5 were used. This doesn't mean the hashes are "uncrackable," but LastPass at least did their due dilligence in using one of the best hashing mechanisms available for their passwords. Nonetheless, users are urged to replace their master password for LastPass, and to update their passwords on other services if they reused that password elsewhere.
```
Comment (catlook) [inline _posts/2015-06-22-this-week-in-infosec-jun-22.md]: ... password for LastPass and to update their ...

File: _posts/2015-06-22-this-week-in-infosec-jun-22.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+# Security Research
+
+- [__Stealing Keys from PCs using a Radio__][radio] - Researchers from Tel Aviv University have released research showing that RSA and ElGamal keys can be recovered using simple, cheap hardware to detect and analyze electromagnetic signals produced by the decryption of a crafted ciphertext. The researchers were able to capture these signals from 50cm away using relatively inexpensive equipment, and believe they could increase the range with improved hardware. They constructed a system they call "PITA" to capture signals and store them on an SD card for later decryption, which is small enough to fit inside a small piece of pita bread. A PDF of their technical paper with more details can be found [here][radio2].
```
Comment (catlook) [inline _posts/2015-06-22-this-week-in-infosec-jun-22.md]: ... inexpensive equipment and believe they ...

File: _posts/2015-06-22-this-week-in-infosec-jun-22.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
...ttack, due to the absence of transport layer encryption or cryptographic signatures for packages when downloading language updates. The only protection is a SHA1 checksum provided in a manifest file that is downloaded (without transport encryption) to tell the vulnerable phone which version of the language package to download based on the model of the phone. This is easily bypassed by replacing the legitimate SHA1 value in the manifest with the SHA1 checksum for the attacker's malicious payload. The attack can be carried out with little or no active user interaction, and is executed when the phone is restarted. Users are vulnerable regardless of whether or not they use the Swift keyboard by default, and there is no way to disable or uninstall the keyboard. [Samsung's blog post on the subject][samsung2] claims that the KNOX security platform bundled with many recent Samsung phones will mitigate the issue via kernel protections (though this would seem to be contradicted by the video below), and that Samsung will release patches for the issue over-the-air via KNOX, though no dates are included and releases will likely vary by device and carrier. Here's a video of the attack in action:
```
Comment (catlook) [inline _posts/2015-06-22-this-week-in-infosec-jun-22.md]: ... user interaction and is executed ...
... video below).  The blog also claims that Samsung ...

File: _posts/2015-06-22-this-week-in-infosec-jun-22.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+[fonts3]: https://technet.microsoft.com/library/security/MS15-044
+[fonts4]: https://helpx.adobe.com/security/products/acrobat/apsb15-10.html
+
+[crypto101]: https://www.crypto101.io/
+[crypto101book]: https://9d0df72831e4b345bb93-4b37fd03e6af34f2323bb971f72f0c0d.ssl.cf5.rackcdn.com/Crypto101.pdf
+[crypto101repo]: https://github.com/crypto101
+
+[commandline]: https://github.com/jlevy/the-art-of-command-line
+[bash]: https://www.gnu.org/software/bash/
+[percol]: https://github.com/mooz/percol
```
Comment (catlook) [inline _posts/2015-06-22-this-week-in-infosec-jun-22.md]: [another nice selection of tidbits! - Cat]

**Comment by cneill** [comment]
Thanks, Cat! Just updated with the suggested edits. One of these days I will learn to use the comma properly, I swear!

**Pull request merged**